core/Twig.php

```diff
@@ -57,6 +77,11 @@ function piwik_escape_filter(Environment $env, $string, $strategy = 'html', $cha
     }
 }
 
+/**
+ * @param string|number $amount
+ * @param int $idSite
+ * @return mixed
```

Comment:
Does this should aways be a string?

```suggestion
 * @return string
```

Reply:
Can't easily change that, as `formatCurrency` has no correctly defined return types, therefor it would mark it as violation here.